test/modules/map/components/contextMenu/MapContextMenuContent.test.js

```diff
@@ -93,6 +116,9 @@ describe('OlMapContextMenuContent', () => {
 			expect(stringifyMock).toHaveBeenCalledWith(coordinateMock, GlobalCoordinateRepresentations.WGS84);
 			expect(elevationMock).toHaveBeenCalledOnceWith(coordinateMock);
 			expect(administrationMock).toHaveBeenCalledOnceWith(coordinateMock);
+
+			expect(element.shadowRoot.querySelectorAll('ba-share-position-chip')).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('ba-share-position-chip')[0].getModel().center).toBe(coordinateMock);
```

Comment:
No, here we test the presence/correctness of the `center`  property. We do not want/need to access the internal model.